roles/ceph-iscsi-gw/templates/tcmu-runner.service.j2

```diff
@@ -29,6 +29,7 @@ ExecStart=/usr/bin/{{ container_binary }} run --rm \
   -v /lib/modules:/lib/modules \
   -v /etc/ceph:/etc/ceph \
   -v /var/log/ceph:/var/log/ceph:z \
+  -v /var/log/tcmu-runner:/var/log/tcmu-runner:z \
```

Comment:
Could we just bind mount the default log file ?

And I think wa can remove the `/var/log/ceph` bind mount

Reply:
> Could we just bind mount the default log file ?

so we logrotate a file which is bind-mounted by podman?

Reply:
looks like it's not possible so let's continue with the directory